test/mock/frames/recursive.html

```diff
@@ -21,6 +21,6 @@
 </head>
 <body>
 	<h1>Recursive iframe</h1>
-	<iframe src="recursive.html">
+	<iframe src="nested1.html">
```

Comment:
What is the nature of this change? It would help to know as I'm learning more about recursive iframes.